plugins/discourse-details/plugin.rb

```diff
@@ -37,7 +37,7 @@
       .each do |el|
         text = el.css("summary").text
         link = fragment.document.create_element("a")
-        link["href"] = post.url if post
+        link["href"] = post&.url.presence || Discourse.base_url
```

Comment:
We always want a valid `href` so if there's no post associated, just link to the Discourse instance.